### PR TITLE
[tst] added integration test for removedfiles inspection

### PR DESCRIPTION
**Pull request by pr-watson**
Signed-off-by: prwatson <prwatson@redhat.com>

### CODE REVIEW

**Review by dcantrell — CHANGES_REQUESTED** [review]
Couple of things to do:

- There's no need to duplicate the tests for all of the compressed file types.  Those tests (wherever they were) were meant to test decompression support in rpminspect.  It's fine to just use the first block, like gzip, and forget the rest.
- Likewise, it's not necessary to use the gzip sample files here.  The removedfiles inspection just needs a file, so I usually just use the rpminspect executable or a text file.  Look at some of the other inspections for an example.  But it's fine to use the gzip files too, it just may be confusing to future readers.
- Drop the 'Gzip' prefix on the class names since that's not what these tests are about.
- You need to run 'make flake8' and 'make black' before submitting a PR for the test suite.  Those need to pass and if they don't, you need to reformat the Python code.
- Also, make sure the copyright and Authors list is correct at the top of the file.

File: test/test_removedfiles.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
+#
+# Copyright © 2020 Red Hat, Inc.
+# Author(s): David Cantrell <dcantrell@redhat.com>
```
Comment (jimbair) [inline test/test_removedfiles.py:3]: You'll definitely want to update the copyright and author(s) section to reflect 2022 and you =)

Comment (pr-watson) [reply test/test_removedfiles.py:3]: Adding that in the upcoming PR thanks

**Comment by pr-watson** [comment]
Added test_removedfiles.c containing test cases for TestCompareRPMS, TestCompareKoji, and security files. The tests check that the result and waiver_auth are as expected when a file is removed and not removed.

**Comment by dcantrell** [comment]
This PR includes a conflicting change that was already committed as 20ba55597c90d1bf335e4ad0fd54b6c9ecf7c229.  Please remove it.

**Comment by dcantrell** [comment]
PR#887 replaces this PR.

**Pull request closed without merging**